### ISSUE TITLE
Maintenance support

### ISSUE THREAD

**Issue by MiguVT** [Elytrium/FastMOTD#38]
add support to https://hangar.papermc.io/kennytv/Maintenance/versions pls 💖
Make MOTD handle by Maintenance or something like it for custom-motd when maintenance
The maintenance from FastMOTD dont have every option that production servers need.

**Comment by SaltedYuY** [comment]
Yes, in the latest version, there are still many issues with the maintenance mode, and I hope it can be improved

**Comment by MiguVT** [comment]
And what about adding support for maintenance plugin??
